xla/debug_options_flags.cc

```diff
@@ -166,9 +166,9 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
   opts.set_xla_gpu_enable_highest_priority_async_stream(true);
 
   opts.set_xla_gpu_enable_pipelined_collectives(false);
```

Comment:
Can we document what is the ration of that flags with the one bellow?
When each case is really on?

Reply:
Would updating the flag help display string be enough (https://github.com/openxla/xla/blob/2b0f417b5fe983b0e30e7982e077ffa1a7a70852/xla/debug_options_flags.cc#L1446-L1451) or do you want additional comments near these lines?

Reply:
Updating the flags string description sound good to me.